data/docs/migration/migrate-from-grafana.mdx

```diff
@@ -0,0 +1,271 @@
+| Gauge          | ✓       | -       | Similar functionality available via [Value panels](/docs/dashboards/panel-types/value/) with thresholds                                |
+| Node Graph     | ✓       | -       | [Service Map](/docs/userguide/service-map/) provides similar functionality for service dependencies                             |
+| Logs           | ✓       | ✓       | Dedicated [List panel type](/docs/dashboards/panel-types/list/) for logs visualization                                               |
+| Traces         | ✓       | ✓       | Advanced trace visualization through [APM features](/docs/userguide/traces/)                                              |
+| Flame Graph    | ✓       | ✓       | Both support flame graphs for performance profiling                                            |
+| State Timeline | ✓       | -       | Not currently supported                                                                        |
+| Status History | ✓       | -       | Not currently supported                                                                        |
+| Candlestick    | ✓       | -       | Not currently supported                                                                        |
+| XY Chart       | ✓       | -       | Not currently supported                                                                        |
+| Geomap         | ✓       | -       | Not currently supported                                                                        |
+| Canvas         | ✓       | -       | Not currently supported                                                                        |
+| News           | ✓       | -       | Not currently supported                                                                        |
+
+SigNoz does not currently support plugins, so in case you are using any plugins that provide additional panel types in your Grafana, you will need to find an alternative or recreate the same functionality using SigNoz's features.
+
+Please refer to [Panel Types](/docs/dashboards/panel-types/) documentation for more details on the panel types supported by SigNoz.
+
+## Migrating Alerts
+
+While migrating your observability stack from the LGTM stack to SigNoz, you'll also need to recreate your alerts in SigNoz's alert management system.
+
+### Setting Up Notification Channels
+
+Before migrating your alerts to SigNoz, you will need to set up the notification channels you want to use. Below is a list of notification channels supported by AlertManager/Grafana and SigNoz:
+
+| Notification Channel | AlertManager/Grafana | SigNoz             |
+|----------------------|----------------------|-------------------|
+| Email                | ✓                    | ✓                 |
+| Slack                | ✓                    | ✓                 |
+| Microsoft Teams      | ✓                    | ✓ (Paid plans)    |
```

Comment:
Microsoft teams is not available in free plans as well

Reply:
Sorry, could you clarify what you mean by "not available in free plans as well"? That's the reason I explicitly mentioned `(Paid plans)`.

Or do you meant "now available" instead of "not available"? If so, we should update https://signoz.io/docs/setup-alerts-notification/#type as well.

Reply:
Sorry I meant "Now" available

Reply:
Cool, removed `(Paid plans)`